tests/erica_legacy/elster_xml/test_elster_xml_generator.py

```diff
@@ -984,7 +985,7 @@ def test_add_vast_antrag_nutzdaten(self):
         xml_string = _pretty(xml_top)
 
         self.assertEqual('false', xml_top.find('.//SpezRechtAntrag/Veranlagungszeitraum/Unbeschraenkt').text)
-        self.assertEqual(str(TEST_EST_VERANLAGUNGSJAHR), xml_top.find('.//SpezRechtAntrag/Veranlagungszeitraum/Veranlagungsjahre/Jahr').text)
+        self.assertEqual(str(self.valid_user_data['tax_year']), xml_top.find('.//SpezRechtAntrag/Veranlagungszeitraum/Veranlagungsjahre/Jahr').text)
```

Comment:
Isn't that the test you are looking for?

Reply:
Kind of - without the API part. But close enough. :)